site/guides/producer/functions/ts/README.md

```diff
@@ -37,6 +37,6 @@ This provides various advantages:
 - Read the complete [Typescript Developer Guide].
 - Learn how to [run functions].
 
-[Typescript Quickstart]: quickstart/
-[Typescript Developer Guide]: develop/
-[run functions]: ../../../consumer/function/
+[Typescript Quickstart]: guides/producer/functions/ts/quickstart/
```

Comment:
How does this work? It's relative link.

Reply:
Relative pathing is deactivated in Docsify's settings so these should all be prefixed with a forward slash to make that clearer.